test/e2e/Tests/Fixtures/FixtureHelpers.cs

```diff
@@ -100,12 +100,14 @@ internal static void AddDurableBackendEnvironmentVariables(Process funcProcess,
                 funcProcess.StartInfo.EnvironmentVariables["AzureFunctionsJobHost__extensions__durableTask__storageProvider__connectionStringName"] = "SQLDB_Connection";
                 funcProcess.StartInfo.EnvironmentVariables["AzureFunctionsJobHost__extensions__durableTask__storageProvider__createDatabaseIfNotExists"] = "true";
                 funcProcess.StartInfo.EnvironmentVariables["AzureFunctionsJobHost__extensions__durableTask__MaxGrpcMessageSizeInBytes"] = "6291456";
+                funcProcess.StartInfo.EnvironmentVariables["AzureFunctionsJobHost__extensions__durableTask__ThrowStatusExceptionsOnRaiseEvent"] = "true";
```

Comment:
Why are we setting this only on a subset of backends (instead of all backends)?

Reply:
Only Azure Storage sets [CheckStatusBeforeRaiseEvent](https://github.com/Azure/azure-functions-durable-extension/blob/dev/src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProvider.cs#L61C30-L61C57) to true. SQL and DTS do not implement this setting, so their default is false. So at these two backends, no exception will be thrown unless the setting is explicitly specified.